### PR TITLE
Enhancement: Boxes Indexer Integration test

### PR DESCRIPTION
**Pull request by ahangsu**

### CODE REVIEW

File: .test-env
Comment (ahangsu) [inline .test-env:3]: TODO: once algorand-sdk-testing PR is merged, do
```suggestion
SDK_TESTING_BRANCH="feature/box-storage"
```

File: tests/cucumber/integration.tags
Comment (tzaffi) [inline tests/cucumber/integration.tags:5]: this tag may go away?

**Review by tzaffi — COMMENTED** [review]
Basically approvable but waiting on decision for future of tag `@applications.boxes_indexer_confirmed`

File: tests/cucumber/integration.tags
Comment (ahangsu) [reply tests/cucumber/integration.tags]: gone

**Review by tzaffi — APPROVED** [review]
LGTM

**Comment by tzaffi** [comment]
> LGTM

LEBTM

File: tests/cucumber/steps/steps.js
Comment (algochoi) [inline tests/cucumber/steps/steps.js:4463]: I think you can `assert.fail` here and in the other two assertions below:

```
assert.fail(
          `expecting algod or indexer, got ${fromClient}`
        );
```

**Review by algochoi — COMMENTED** [review]
Minor comment

File: tests/cucumber/steps/steps.js
Comment (algochoi) [inline tests/cucumber/steps/steps.js:4535]: Nice to unify here: ```assert.fail(`expecting algod or indexer, got ${fromClient}`);```

File: tests/cucumber/steps/steps.js
Comment (algochoi) [inline tests/cucumber/steps/steps.js:4560]: ```assert.fail(`expecting algod or indexer, got ${fromClient}`);```

**Review by algochoi — COMMENTED** [review]
LGTM - just a reminder for the other `assert.fail`s

File: tests/cucumber/steps/steps.js
Comment (ahangsu) [reply tests/cucumber/steps/steps.js]: good find, my trash code on js

File: tests/cucumber/steps/steps.js
Comment (ahangsu) [reply tests/cucumber/steps/steps.js]: good find, my trash code on js

**Review by algochoi — APPROVED** [review]

**Pull request merged**
